SoftwareDesign.md

```diff
@@ -0,0 +1,6 @@
+### Software Design ###
+A passion of mine since I was very young, software design to me is the art of creating enteractive applications that create solutions to issues that people encounter everyday. From the basic store front website to the informative blog, we write applications that are solutions to issues we all encounter. To me this is like seeing someone have a dream, and I am like a like a wish granter, and can create that dream. 
+
```

Comment:
I'm glad that you're passionate about design. Nobody likes a poorly designed website/app.